packages/back-end/src/types/Integration.ts

```diff
@@ -70,16 +117,41 @@ export type FactMetricData = {
   capCoalesceCovariate: string;
   capCoalesceDenominatorCovariate: string;
   minMetricDelay: number;
-  raMetricSettings: {
-    hours: number;
-    minDelay: number;
-    alias: string;
-  };
+  raMetricSettings: FactMetricRegressionAdjustmentSettings;
   metricStart: Date;
   metricEnd: Date | null;
   maxHoursToConvert: number;
 };
 
+export type FactMetricSourceData = {
+  factTable: FactTableInterface;
```

Comment:
question: Should this be an array? As a metric can refer to 2 tables?

Not for now, but relates to the cross-fact table metrics.